docs/DEPLOYMENT.MD

```diff
@@ -17,5 +16,7 @@ This document covers steps on setting up this repository on various cloud hostin
      ```
 8. To deploy your code on to staging, use `npm run deploy-staging/yarn deploy-staging`.
 
-# Production
-Setting up a production environment on AWS is same as staging.
+### Production
+Setting up a production environment on AWS is the same as setting up a staging environment.
+
+**Make sure to setup `.env.staging` for staging environment and `.env.production` for production environment.**
```

Comment:
I think the script for `deploy` should be in this section and the staging section should just contain the script for `deploy-staging` no?

Reply:
Pritam's making some changes to the documentation.

Reply:
I'm just correcting stuff lol. Don't even know what @deepansh96 is talking about, so please make the changes he's suggesting